### ISSUE TITLE
draw.io bug - saving the file acts differently when in a group

### ISSUE THREAD

**Issue by adamfk** [StateSmith/StateSmith-drawio-plugin#25]
Needs a draw.io ticket to be opened. Make sure to add lots of detail with easy to reproduce steps. The draw.io team doesn't put up with low effort issue submissions.

In diff below, left side is normal file. Right side is when file is saved again, but when currently inside another group.

![image](https://user-images.githubusercontent.com/274012/218335327-d00dfda9-35d2-4079-9db0-4f53fa4ab5f9.png)

**Comment by adamfk** [comment]
Upstream drawio bug is fixed. It may still take a while before the vscode extension updates to the fixed version though.

**Comment by emmby** [comment]
It's been a year, is this still an issue with the vscode extension?

**Comment by adamfk** [comment]
> It's been a year, is this still an issue with the vscode extension?

Yeah, still an issue in version 1.6.6 (the latest). It's been over a year since the last release of the vscode extension. https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio

The draw.io offline application is generally more stable. It's not quite as convenient as vscode extension, but it only takes a few clicks (every time you launch it) to get the StateSmith plugin working: https://github.com/StateSmith/StateSmith-drawio-plugin/wiki/Use-with-app-or-website
